### PR TITLE
Generate meaningful names for themes and add white & black buttons

### PR DESCRIPTION
**Pull request by heokhe**

### CODE REVIEW

File: handlers/cbquery.js [hunk 1]
```diff
@@ -77,7 +121,7 @@ module.exports = bot => {
                     type: `document`,
                     media: {
                         source: Buffer.from(completedTheme, `binary`),
-                        filename: `${name} by @${process.env.BOT_USERNAME}.${data}`,
+                        filename: `${name} - @${process.env.BOT_USERNAME}.${data}`,
```
Comment (vrumger) [inline handlers/cbquery.js]: I prefer `by @ThemerBot`.

Comment (heokhe) [reply handlers/cbquery.js]: I'm too lazy to revert it.

Comment (vrumger) [reply handlers/cbquery.js]: I'm too lazy to merge it. ;D

Comment (heokhe) [reply handlers/cbquery.js]: :expressionless:

**Pull request merged**